moveos/moveos-stdlib/moveos-stdlib/sources/object.move

```diff
@@ -530,6 +573,60 @@ module moveos_std::object {
         object_entity.size
     }
 
+    // ===== Original raw_table APIs =====
+    // TODO refactor later
+
+    /// Add a new entry to the table. Aborts if an entry for this
+    /// key already exists. The entry itself is not stored in the
+    /// table, and cannot be discovered from it.
+    public(friend) fun raw_table_add<K: copy + drop, V>(table_handle: ObjectID, key: K, val: V) {
```

Comment:
Does the `raw_table` prefix need to be renamed?

Reply:
To avoid function name conflict, add raw_table prefix, which will be refactored later.